src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -50,11 +51,23 @@ public CommandResult execute() {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
             final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            if (!Collections.disjoint(getLowerCase(wordsInName), getLowerCase(keywords))) {
                 matchedPersons.add(person);
             }
         }
         return matchedPersons;
     }
 
+    /**
+     * Converts words to lowercase.
+     *
+     * @param keywords to change to lowercase
+     * @return list of words in lowercase
+     */
+    private Set<String> getLowerCase(Set<String> keywords) {
+        return keywords.stream()
+                .map(String::toLowerCase)
+                .collect(Collectors.toSet());
+    }
```

Comment:
Good job with using Java 8 stream to convert all elements of a collection to uppercase